uwb-beacon-firmware/src/ranging_thread.c

```diff
@@ -103,6 +105,10 @@ static void ranging_thread(void *p)
 
     static uint8_t frame[64];
 
+    int current_anchor_mac_index = 0;
+    uint16_t anchor_macs[] = {7, 10, 11, 14};
```

Comment:
Hardcoded anchor mac addresses, we need to do it properly

Reply:
You want it done in this PR or in another one?

Best way is probably just to dump it in the parameter tree. Since we don't have support for lists of integers, that would need to be implemented as well. Or to use strings and parsing, e.g. storing "7;10;11;14" but I find it a bit ugly.

Reply:
let's address it later